Content.Client/Options/UI/Tabs/MiscTab.xaml.cs

```diff
@@ -36,12 +37,20 @@ public MiscTab()
             layoutEntries.Add(new OptionDropDownCVar<string>.ValueOption(layout.ToString()!, Loc.GetString($"ui-options-hud-layout-{layout.ToString()!.ToLower()}")));
         }
 
+        // EE - Chat stacking options for how far back in the chat to stack.
+        var chatStackEntries = new List<OptionDropDownCVar<int>.ValueOption>();
+        chatStackEntries.Add(new OptionDropDownCVar<int>.ValueOption(0, Loc.GetString("ui-options-chatstack-off")));
```

Comment:
```suggestion
        var chatStackEntries = new List<OptionDropDownCVar<int>.ValueOption>
        {
            new(0, Loc.GetString("ui-options-chatstack-off")),
            new(1, Loc.GetString("ui-options-chatstack-single")),
            new(2, Loc.GetString("ui-options-chatstack-double")),
            new(3, Loc.GetString("ui-options-chatstack-triple"))
        };
```

Reply:
use a for loop even

Reply:
I don't think you can make this shorter with a for loop

Reply:
have the off/single/double/triple be in ftl select expression so its not hardcoded loc strings for every number